src/launch/java/baritone/launch/mixins/MixinItemStack.java

```diff
@@ -50,7 +51,15 @@ private void recalculateHash() {
             at = @At("RETURN")
     )
     private void onInit(CallbackInfo ci) {
+        if (ItemStackInitGuard.isEntered()) {
+            return;
+        }
+        ItemStackInitGuard.enter();
+        try {
         recalculateHash();
+        } finally {
```

Comment:
```suggestion
        try {
            recalculateHash();
        } finally {
```
Slight formatting error.